iree/tools/opt_main.cc

```diff
@@ -17,13 +17,13 @@
 // Based on mlir-opt but without registering passes and dialects we don't care
 // about.
 
+#include "init_dialects.h"
```

Comment:
In the future please include the directory path for includes. Just had to do some fixup internally to deal with lint. Not a huge deal :-)